app/__init__.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, render_template, redirect, url_for, request
+from .forms.DeployForm import DeployForm
```

Comment:
Which reminds me I wanted to set up a travis test that runs pep8 on the code. Adding to issues... Thanks!